signature/docker_test.go

```diff
@@ -44,6 +54,57 @@ func TestSignDockerManifest(t *testing.T) {
 	assert.Error(t, err)
 }
 
+func TestSignDockerManifestWithPassphrase(t *testing.T) {
+	defer killGPGAgent(t)
```

Comment:
Shouldn’t the test just do it at the _start_? That way it would work from a known state.

Reply:
Sure, that sounds much cleaner.